android/app/src/main/cpp/jitter_buffer.cpp

```
@@ -0,0 +1,144 @@
+    // link-quality reporter sees the overflow and may decide to step bitrate
+    // down. We could equivalently drop the oldest queued frame, but dropping
+    // the newest preserves decoder context (PLC works better when contiguous
+    // audio is preserved) and keeps the playhead's chronology intact.
+    if (frames_.size() >= audio_config::kJitterMaxDepth) {
+        ++lateCount_;
+        return false;
+    }
+
+    // Insert in modular-sorted order. Working window is small (<= kMaxDepth
+    // = 10 frames) so a linear scan is faster than a tree.
+    auto it = frames_.begin();
```

Clang (14.0.6)

[warning] variable name 'it' is too short, expected at least 3 characters

(readability-identifier-length)